hieradata/development.yaml

```diff
@@ -107,7 +107,6 @@ govuk::apps::authenticating_proxy::mongodb_nodes: ['localhost']
 govuk::apps::authenticating_proxy::govuk_upstream_uri: 'http://government-frontend.dev.gov.uk'
 govuk::apps::backdrop_write::enable_procfile_worker: false
 govuk::apps::collections_publisher::enable_procfile_worker: false
-govuk::apps::contacts::extra_aliases: ['contacts-frontend-old', 'contacts-admin']
```

Comment:
Yup, I thought it was aliasing one to the other but it's actually defining two aliases. I've added it back.